### DISCUSSION TITLE
Ready for rc4?

### DISCUSSION THREAD

**Discussion by fballiano** [OpenMage/magento-lts#3250] [Releases]
Hello people, do you think we're ready for a rc4 of the new releases?

also, what do you think it's missing from a final release?

**Comment by sreichel** [comment]
I know you hate me for this ...

.. i'd check all PRs for new methods to have correct type hints set. 

See https://github.com/OpenMage/magento-lts/discussions/2748

**Reply by fballiano** [reply]
for the final release? the problem that I see is that it is an extremely long task

**Reply by sreichel** [reply]
As longer we wait, it will become harder. So, YES.

It just needs to review _a few_ PRs again :P

**Reply by fballiano** [reply]
but the tasks of #2748 were not even started right?

**Reply by sreichel** [reply]
We did. In a little. There a few new methods that use type hints.

I'd _enforce_ this ASAP.

For `next` branch i'd allow to add return types to existing mehtods, too.

**Reply by fballiano** [reply]
ok but dosn't seem something related to releasing a new version.

IMHO the issue/pr templates are a bit outdated and we should have that info there
